tests/wp-tests-bootstrap.php

```diff
@@ -38,17 +38,6 @@ function _remove_automated_checks() {
 	_remove_automated_checks();
 } );
 
-/**
```

Comment:
Ah sorry, was just being cheeky, probably makes sense to explicitly set a timezone so that all date based assertions stay consistent. This can stay in.

Reply:
maybe you're being cheeky now 🤔  hehe ok.

Reply:
You just never know 😄